Lesson_Materials/In_Order_Queue/index.html

```diff
@@ -155,40 +151,74 @@
 					<div class="container">
 						<div class="col">
 							<code class="code-100pc"><pre>
-auto devicePtr   = usm_wrapper&lt;int&gt;(
-  malloc_device&lt;int&gt;(1024, inOrderQueue));
-  
-  inOrderQueue.memcpy(devicePtr, data, sizeof(int));
-  
-  inOrderQueue.parallel_for&lt;kernel_a&gt;(sycl::range{1024},
-  [=](sycl::id&lt;1&gt; idx){
-  devicePtr[idx] = /* some computation */
-});
-
-inOrderQueue.parallel_for&lt;kernel_b&gt;(sycl::range{1024},
-  [=](sycl::id&lt;1&gt; idx){
-  devicePtr[idx] = /* some computation */
-});
-
-inOrderQueue.memcpy(data, devicePtr, sizeof(int));
-
-<mark>inOrderQueue.wait();</mark>
+sycl:: queue Q{sycl::property::queue::in_order};
+for (auto task: tasks) {
+  Q.submit(task);
+}
+Q.wait();
+							</code></pre>
+						</div>
+						<div class="col" data-markdown>
+							* All events are executed serialy
+							* Ease of programing (no race condition)
+							* Zero concurency / Potential suboptimal performance
+						</div>
+					</div>
+				</section>
+				<section>
+					<div class="hbox" data-markdown>
+						#### Using an out-of-order queue (USM): But how to handle concurency
+					</div>
+					<div class="container">
+						<div class="col">
+							<code class="code-100pc"><pre>
+std::vector<sycl:: queue> Qs = map_queues_to_task();
+for (auto [Q,task]: queue_tasks) {
+  Q.submit(task);
+}
+for (auto [Q,task]: queue_tasks) {
+  Q.wait();
+}
 							</code></pre>
 						</div>
 						<div class="col" data-markdown>
-							* In the USM model dependencies are greatly simplified.
-							* It's no longer necessary to chain the commands together with `event` as they execute in the order they are enqueued.
-							* Simply calling `wait` on the the `queue` is sufficient.
+							* Manual Scheduling (painfull to extract full concurency)
+							* When correcly done, acheive concurency between queues;
 						</div>
 					</div>
 				</section>
 				<!--Slide 8-->
+				<section>
+					<div class="hbox" data-markdown>
+						## Note on buffer/accessors
+						* Best of Both world
+						* Buffer / Accessors handle the dependency for you (no need for `depend on`)
+						* Hence, using in or out of order queues will not change the scemantic of your program!
```

Comment:
```suggestion
						* Hence, using in or out of order queues with buffer/accessors will not change the semantics of your program!
```
Just to clarify that this is only true for buf acc